spec/models/student_account_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe StudentAccount, type: :model do
+
+  context 'associations' do
+    it { is_expected.to belong_to :user }
+    it { is_expected.to have_many :day_records }
+    it { is_expected.to have_many :closures }
+  end
+
+  context 'validations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.